### ISSUE TITLE
GC2000 debug tool

### ISSUE THREAD

**Issue by nmjohnson** [mpmc/HS602#12]
Seems like there is a "debug tool" for the GC2000 available here: https://www.dmmdownload.com/downloads/CURRENT/Testing/

https://www.dmmdownload.com/downloads/CURRENT/Testing/GC2000_DebugTool.zip

**Comment by gingerbeardman** [comment]
Seems to output a log file.

![DebugTool-2](https://user-images.githubusercontent.com/49612/215355650-6b7af720-5cce-422c-a7c1-d1c4e69e4276.png)

**Comment by nmjohnson** [comment]
yeah my guess it is was used by customer service to help resolve issues, but maybe it can be useful for RE
